### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_Notebooks - Run_Keyboard:"Filter" button in Results Table is not accessible using keyboard.

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio-mysql#186]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #GH-MySQLextensionforAzureDataStudio-Win32-Mar23; #MySQL extension for Azure Data Studio; #Win32; #WCAG2.1.1; #Keyboard; #Benchmark; #External; #External:AzureDataStudio; #FirstParty:22657;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Open an existing notebook that has code cells in it.
3. Click Run Cells on Notebook toolbar.
4. Tab till Filter
5. Verify that Keyboard focus is not shifting to Filter
### Actual Result:
"Filter" button in Results Table is not accessible using keyboard.
### Expected Result:
Filter button in Results Table should be accessible using keyboard.
### Attachment:

![Video0 (6)](https://user-images.githubusercontent.com/100189407/228870875-9e5a418e-5698-41f9-be78-a74b6a7d2137.gif)

**Comment by nitishsharma25** [comment]
This issue is external to MySQL extension as it is a common UI of Azure Data Studio.

**Comment by VijayaBharathyA** [comment]
External bug :https://github.com/microsoft/azuredatastudio/issues/22657
Could You please Reopen the bug since We have follow External process for this bug We will act on this bug once external bug is fixed and closed. Please don't close the bug

**Comment by VijayaBharathyA** [comment]
Version: 1.44.0-insider (user setup)
Commit: 6a2ec12a3537725aaea52716c734114504760118
Date: 2023-04-07T05:32:37.320Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Since we are able to access filter using F3 hence closing the bug
![Video0](https://user-images.githubusercontent.com/100189407/232376755-82cadce6-4a3c-4dff-853f-41e4bf5800d8.gif)
